terraform-common.mk

```diff
@@ -47,4 +47,4 @@ graph:
 	terraform graph -draw-cycles | dot -Tpng > graph.png
 
 $(ENV_NAME).tfvars:
-	$(TOP)/bin/generate-tfvars >$@
+	$(TOP)/bin/generate-tfvars $@
```

Comment:
Not a bug that was introduced in this PR, but `$@` should be `"$@"`, otherwise paths with spaced in them will break.